src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,39 @@
+package duke.task;
+
+/**
+ * Represents a task with an event.
+ */
+public class Event extends Task {
+
+    protected String at;
```

Comment:
Similarly, I think you can consider using dateAt instead of at so it's clearer what the variable is referring to.